api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -4914,3 +4960,230 @@ message OktaAssignmentResult {
     (gogoproto.jsontag) = ""
   ];
 }
+
+// AccessListCreate is emitted when an access list is created.
+message AccessListCreate {
+  // metadata is common event metadata
+  Metadata metadata = 1 [
```

Comment:
Just pointing out inconsistency w.r.t capitalization. `AccessListMemberMetadata` capitalizes field names, but down here they're all lower case.

I'm still of the opinion that unless there's a very good reason to break with the existing conventions, we should stay consistent.

Reply:
Ah snap, that wasn't intentional. I'll fix these!